tests/functional/testSubscription.js

```diff
@@ -105,8 +177,8 @@ describe('Subscription manager', function() {
          await vpn.clickOnQuery(queries.screenHome.CONTROLLER_TOGGLE.visible());
 
          // Step 3: Verify that user gets the "Subscribe to Mozilla VPN" screen.
-         await vpn.waitForQuery(
-             queries.screenHome.SUBSCRIPTION_NEEDED.visible());
+         await vpn.waitForQuery(queries.screenSubscriptionNeeded
```

Comment:
I have noticed that this test is missing a Step 2!

Please clean this up while you're here.

I think the new test you've written would be best placed after this test because it's kinda adding more building blocks to the same starting point.

If there's a reason they're ordered this way, let me know.